CONTRIBUTING.md

```diff
@@ -75,7 +75,7 @@ reference GitHub issues that this commit **Closes**.
 
 | Task | Description |
 |------|-------------|
-| `grunt jshint` | run JSHint on client side and testing JavaScript. |
+| `grunt eslint` | run ESLint on client side and testing JavaScript. |
```

Comment:
Maybe add a new section `grunt lint` that talks about runing all the linters.

We have a few missing lint related tasks from this section: `jsonlint`, `jscs` and `amdcheck`